decidim-forms/app/types/decidim/forms/questionnaire_type.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Forms
+    QuestionnaireType = GraphQL::ObjectType.define do
```

Comment:
I seen this very same files for decidim-forms in other dependant PRs, is it normal?

Reply:
Because it was depending on meetings. It is solved now.